Simplified/Reader2/NYPLReaderExtensions.swift

```diff
@@ -46,20 +46,25 @@ extension NYPLBook {
 
 @objc extension NYPLRootTabBarController {
   func presentBook(_ book: NYPLBook) {
-    guard let libModule = r2Owner?.libraryModule else {
+    guard let libraryService = r2Owner?.libraryService, let readerModule = r2Owner.readerModule else {
```

Comment:
Missing the optional sign on the second `r2Owner`
Maybe we should do a check on the `r2Owner` instead of the `libraryService` and `readerModule` since those are non-optional

Reply:
fixed